packages/bff/package.json

```diff
@@ -14,8 +14,9 @@
     "typeorm:run": "typeorm-ts-node-commonjs migration:run -- -d src/data-source.ts",
     "typeorm:revert": "npm run typeorm migration:revert",
     "dev": "concurrently \"npx tsc --watch\" \"nodemon --watch 'src/**/*.ts'\"",
+    "docker": "concurrently \"npx tsc --watch\" \"nodemon --watch '/app/packages/bff/src/**/*.ts'\"",
```

Comment:
Do we need concurrently? Is it not enough to run `ts-node`? Nodemon should be able to work together with `ts-node`.

Reply:
If you know how to do it without, then of course we can